src/main/java/uk/gov/hmcts/reform/sscs/robotics/json/RoboticsJsonMapper.java

```diff
@@ -76,7 +76,12 @@ public JSONObject map(RoboticsWrapper roboticsWrapper) {
         if (roboticsWrapper.getState().equals(State.READY_TO_LIST)) {
             isReadyToList = "Yes";
         }
+        String isDigital = "No";
```

Comment:
Are these isReadyToList and isDigital same flags?

Reply:
From the Jira ticket. isReadyToList uses the state of the case and determines if its ready to list, it does not check if it's digital.